sdk/search/platform-matrix.json

```
@@ -0,0 +1,6 @@
+{
+    "matrix": {
+      "$IMPORT": "eng/pipelines/templates/stages/platform-matrix.json"
+      "ArmTemplateParameters": "@{ enableCmk = $false }"
+    }
+  }
```

Missing comma and whitespace normalization

```suggestion
{
    "matrix": {
        "$IMPORT": "eng/pipelines/templates/stages/platform-matrix.json",
        "ArmTemplateParameters": "@{ enableCmk = $false }"
    }
}
```